SmartDeviceLink/private/SDLSoftButtonManager.m

```diff
@@ -125,7 +125,7 @@ - (void)setSoftButtonObjects:(NSArray<SDLSoftButtonObject *> *)softButtonObjects
     // Set the soft button ids. Check to make sure no two soft buttons have the same name, there aren't many soft buttons, so n^2 isn't going to be bad
     for (NSUInteger i = 0; i < softButtonObjects.count; i++) {
         NSString *buttonName = softButtonObjects[i].name;
-        softButtonObjects[i].buttonId = i * 100;
+        softButtonObjects[i].buttonId = (i * 100) ? (i * 100) : (i + 1);
```

Comment:
I have no idea why the code increments the `buttonId`s by 100. There is no reason for doing so as far as I can tell. We could probably get away with just doing
`softButtonObjects[i].buttonId = i + 1`;

Reply:
I thought of replacing the first index with 1, in case the *100 in other indexes affects other things. But sure i can increment the index as suggested.